pkg/credentialprovider/plugin/plugin.go

```diff
@@ -56,6 +57,7 @@ var (
 
 	apiVersions = map[string]schema.GroupVersion{
 		credentialproviderv1alpha1.SchemeGroupVersion.String(): credentialproviderv1alpha1.SchemeGroupVersion,
+		credentialproviderv1beta1.SchemeGroupVersion.String():  credentialproviderv1beta1.SchemeGroupVersion,
 	}
 )
 
```

Comment:
/hold

don't you need to register `kubeletconfigv1beta1.AddToScheme(scheme)` below to be able to read the v1beta1 config?

Reply:
if that's the case, make sure there's a test that would have noticed the beta config could not be read

Reply:
doing it here https://github.com/kubernetes/kubernetes/pull/108847/files#diff-11cd16048f2e38e10a534e679ec731eff9dc634b2070260b8ef9337eced47e21R42

It gets registered with component config

Reply:
the type gets registered into the beta version there, but the beta version doesn't get registered into the scheme used to read config files:

```
func init() {
	install.Install(scheme)
	kubeletconfig.AddToScheme(scheme)
	kubeletconfigv1alpha1.AddToScheme(scheme)
}
```

Reply:
Done, 
test as a followup? otherwise I'll add it tomorrow

Reply:
just duplicate the `"config with multiple providers"` test in pkg/credentialprovider/plugin/config_test.go, rename to `"v1beta1 config with multiple providers"`, and change all the versions in the config to v1beta1

Reply:
Done